src/components/ManagerCompletedCard/ManagerCompletedCard.style.ts

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components'
+
+export const CardContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  background: white;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  border-radius: 15px;
```

Comment:
마이페이지에 있는 모든 카드들 border-radius 값이 다른 것 같아요! 피그마 상에서는 20px입니당
```suggestion
  border-radius: 20px;
```

Reply:
디자인전공이시라 세심하시네요 ㅎㅎ 수정하겠습니다!